client-web/src/exlibris-app/exlibris-app.html

```diff
@@ -0,0 +1,200 @@
+
+<link rel="import" href="menu/exlibris-menu-drawer.html">
+
+<dom-module id="exlibris-app">
+    <template>
+        <style>
+            :host {
+                --exlibris-app-primary-color: var(--paper-blue-600);
+                --app-secondary-color: var(--paper-grey-600);
+                --exlibris-app-background-color: var(--paper-grey-200);
+                --app-drawer-width: 250px;
+                display: block;
+            }
+
+            app-drawer {
+                --app-drawer-content-container: {
+                    background-color: var(--exlibris-app-background-color);
+                }
+            }
+
+            app-toolbar {
+                background-color: var(--exlibris-app-primary-color);
+                color: #fff;
+            }
+
+            app-drawer-layout:not([narrow]) [drawer-toggle] {
+                display: none;
+            }
+        </style>
+
+        <app-location route="{{route}}"
+                      use-hash-as-path></app-location>
+
+        <app-route
+                route="{{route}}"
+                pattern="/:page"
+                data="{{routeData}}"></app-route>
+
+        <app-drawer-layout fullbleed narrow="{{narrow}}">
+            <!-- Drawer content -->
+            <app-drawer id="drawer" swipe-open="[[narrow]]" slot="drawer" on-click="_toggleDrawer">
+                <exlibris-menu-drawer selected="[[routeData.page]]"
+                                      count-badges="[[countBadges]]"></exlibris-menu-drawer>
+            </app-drawer>
+            <!-- Main content -->
+            <app-header-layout has-scrolling-region>
+                <app-header slot="header" condenses reveals effects="waterfall">
+                    <app-toolbar>
+                        <paper-icon-button id="number" icon="menu"
+                                           drawer-toggle></paper-icon-button>
+                        <span drawer-toggle>[[appHeaderTitle]]</span>
+                    </app-toolbar>
+                </app-header>
+
+                <iron-pages
+                        selected="[[routeData.page]]"
+                        attr-for-selected="data-page"
+                        fallback-selection="view404"
+                        role="main">
+                    <div data-page="view404">View404</div>
+                </iron-pages>
+            </app-header-layout>
+        </app-drawer-layout>
+    </template>
+
+    <script>
+
+        /**
+         * # Library application. The polymer-based single page app.
+         *
+         * ## How to use `<exlibris-app>` component.
+         * In typical use, just use `<exlibris-app>` at the top of your body:
+         * <body>
+         *   <exlibris-app></exlibris-app>
+         * </body>
+         *
+         * @customElement
+         * @polymer
+         * @appliesMixin ReduxMixin
+         */
+        class ExlibrisApp extends window.ExlibrisApplication.ReduxMixin(Polymer.Element) {
```

Comment:
You can add ExlibrisApp to .eslintrc.json:
```json
"globals": {
  "ExlibrisApp": true
}
```
Then you can use it without window.